tests/migrations/test_operations.py

```diff
@@ -1334,6 +1339,60 @@ def test_alter_unique_together_remove(self):
         operation = migrations.AlterUniqueTogether("Pony", None)
         self.assertEqual(operation.describe(), "Alter unique_together for Pony (0 constraint(s))")
 
+    def test_create_index(self):
+        """
+        Tests the CreateIndex operation.
+        """
+        project_state = self.set_up_test_model("test_crin")
+        Pony = project_state.apps.get_model("test_crin", "Pony")
+        index = models.Index("pink", model=Pony)
```

Comment:
Sorry, I meant: should `model` be removed as an argument of `Index.__init__()` if it can't be used as an argument in normal usage?